tests/UI/pages/BO/catalog/productsV2/add/index.ts

```diff
@@ -163,11 +279,33 @@ class CreateProduct extends BOBasePage {
 
     await pricingTab.setProductPricing(page, productData);
 
-    await this.setProductStatus(page, productData.status);
-
     return this.saveProduct(page);
   }
 
+  /**
+   * Set product status
+   * @param page {Page} Browser tab
+   * @param status {boolean} The product status
+   * @returns {Promise<void>}
+   */
+  async setProductStatus(page: Page, status: boolean): Promise<void> {
+    await this.setChecked(page, this.productActiveSwitchButton(status ? 1 : 0), true);
+  }
+
+  /**
+   * Set product name
+   * @param page {Page} Browser tab
+   * @param name {string} Name of the product
+   * @param locale {string} Locale
+   * @returns {Promise<void>}
+   */
+  async setProductName(page: Page, name: string, locale: string = 'en'): Promise<void> {
+    await this.waitForSelectorAndClick(page, this.productNameLanguageButton);
+    await this.waitForSelectorAndClick(page, this.productNameLanguageDropdownItem(locale));
+    await page.$eval(this.productNameLanguageDropdownItem(locale), (el: HTMLElement) => el.click());
```

Comment:
We should start to get rid of `$eval` as it's discouraged by playwright

Reply:
I agree... but with @nesrineabdmouleh we find only this solution.

When `page.$eval` will be deprecated, we will remove it.

Reply:
Check if this works too, if not, we can let `$eval`
```suggestion
    await page.evaluate(
    (selector) => (document.querySelector(selector) as HTMLElement).click(),
    this.productNameLanguageDropdownItem(locale), 
    );
```

Reply:
Pushed